packages/module/src/css/topology-components.css

```diff
@@ -193,14 +199,20 @@
   --pf-topology__node--m-selected--node__background--Stroke: var(--pf-v5-global--palette--blue-300);
   --pf-topology__node--m-selected--node__label__background--Stroke: var(--pf-v5-global--palette--blue-400);
   --pf-topology__node--m-selected--node__label__background--Fill: var(--pf-v5-global--palette--blue-400);
-  --pf-topology__node--m-selected--separator--Stroke: var(--pf-v5-global--palette--blue-300);
 
   --pf-topology__node--m-selected--node__label__text--Fill: var(--pf-v5-global--Color--100);
   --pf-topology__node--m-selected--m-info--node__label__text--Fill: var(--pf-v5-global--palette--black-900);
   --pf-topology__node--m-selected--m-danger--node__label__text--Fill: var(--pf-v5-global--palette--black-900);
   --pf-topology__node--m-selected--m-warning--node__label__text--Fill: var(--pf-v5-global--palette--black-900);
   --pf-topology__node--m-selected--m-success--node__label__text--Fill: var(--pf-v5-global--palette--black-900);
 
+  --pf-topology__node--m-selected--node__label__text--m-secondary--Fill: var(--pf-v5-global--palette--black-100);
```

Comment:
The secondary label text color doesn't match the primary label, but it does match in the other label status colors and it does in light theme, too. If you wanted it to match, you could update this to `var(--pf-v5-global--Color--100);`. It doesn't look like your other vars are setup this way, but if you wanted the primary/secondary to always match, you could use `var(--pf-topology__node--m-selected--node__label__text--Fill)` here instead.